eng/pipelines/runtime.yml

```diff
@@ -109,7 +109,9 @@ jobs:
       - eng/pipelines/installer/*
     - subset: runtimetests
       include:
+      - src/tests/*
       - src/coreclr/tests/*
```

Comment:
Will we still have `src/coreclr/tests/*` be part of the runtime tests?

Reply:
As I described in the infra standup, my current goal was to make the change as small as possible as the bulk test move makes it hard to manipulate and maintain. As part of this I haven't yet moved everything out of the <code>src/coreclr/tests</code> folder as that would require additional changes to build-test and elsewhere. I plan to do this as a second step, after the tests have been physically moved, and then I'll delete this clause.

Reply:
Ok just checking, makes sense. Thanks for elaborating here.